google/self_link_helpers.go

```diff
@@ -28,6 +28,17 @@ func compareSelfLinkRelativePaths(k, old, new string, d *schema.ResourceData) bo
 	return false
 }
 
+// This method should only be used if the self link references a global resource.
+func compareSelfLinkResourceNames(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
Should the function also be renamed since it's not necessarily comparing self links?